sniffer/sniffer.py

```diff
@@ -33,12 +34,36 @@ def capture_traffic(web_socket: WebSocketApp, web_socket_thread: threading.Threa
     web_socket_thread.join()
 
 
+@yaspin(text="Checking interface mode...")
+def check_inteface_mode():
+    """
+    Checks if the wireless interface has been set to the Monitor mode.
+    """
+    interface_info = subprocess.run(
+        ["iwconfig", f"{DEFAULT_INTERFACE}mon"], capture_output=True, text=True
```

Comment:
What happens when there is no `{DEFAULT_INTERFACE}mon` interface? This needs to be checked.